### PR TITLE
feat(components): action group tight member removes padding

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: libs/components/src/lib/action-group/README.md [hunk 1]
```diff
@@ -74,6 +74,21 @@ Use a `<span>` tag with `role="separator"` for adding separator between the acti
 </vwc-action-group>
 ```
 
+## Tight
```
Comment (rachelbt) [inline libs/components/src/lib/action-group/README.md:77]:
```suggestion
### Tight
```

**Review by rachelbt — COMMENTED** [review]
Looks good :)

**Review by rachelbt — COMMENTED** [review]
regarding this issue:
image
 adding `style="width: 300px;"` to the text-field can solve it :)
 I think we need to set a min-width to. the text-field in another issue :)

File: libs/components/src/lib/action-group/README.md [hunk 2]
```diff
@@ -74,6 +74,34 @@ Use a `<span>` tag with `role="separator"` for adding separator between the acti
...
+  vwc-action-group {
+    inline-size: initial;
+  }
+
+  vwc-text-field {
+    flex-grow: 1;
+  }
+</style>
+
+<vwc-layout column-basis="block" column-spacing="small" style="display: block; max-inline-size: 35ch">
```
Comment (rachelbt) [inline libs/components/src/lib/action-group/README.md:96]:
```suggestion
<vwc-layout column-spacing="small" style="display: block;     --layout-grid-template-columns: 200px 200px;">
```
to get this:
image

File: libs/components/src/lib/action-group/action-group.template.ts [hunk 3]
```diff
@@ -9,12 +9,12 @@ import type { ActionGroup } from './action-group';
...
 
 const getClasses = ({
-	appearance, shape,
+	appearance, shape, tight
 }: ActionGroup) => classNames(
 	'base',
 	[`appearance-${appearance}`, Boolean(appearance)],
 	[`shape-${shape}`, Boolean(shape)],
-
+	['tight', tight],
```
Comment (yinonov) [inline libs/components/src/lib/action-group/action-group.template.ts:17]:
```suggestion
	['tight', Boolean(tight)],
```

File: libs/components/src/lib/action-group/README.md [hunk 4]
```diff
@@ -74,6 +75,34 @@ Use a `<span>` tag with `role="separator"` for adding separator between the acti
 </vwc-action-group>
 ```
 
+### Tight
+
+By default, action group is styled in a spacious manner which visually extends the baseline row size and inline gapping.
+enabling the `tight` member will result in a dense style that fits the "normal" baseline.
```
Comment (yinonov) [inline libs/components/src/lib/action-group/README.md:81]:
```suggestion
enabling the `tight` member will result in a dense style that fits the "normal" components baseline.
```

**Pull request merged**